src/mono/netcore/System.Private.CoreLib/src/System/RuntimeType.Mono.cs

```diff
@@ -78,7 +78,7 @@ public ListBuilder(int capacity)
             {
                 get
                 {
-                    Contract.Requires(index < Count);
+                    Debug.Assert(index < Count);
```

Comment:
We no longer use Contract.* in the libraries or coreclr's corelib, so I replaced all usage in mono's corelib as well.